example-functions.php

```diff
@@ -268,144 +268,148 @@ function yourprefix_register_demo_metabox() {
 		'id'       => $prefix . 'multitaxonomy',
 		'type'     => 'taxonomy_multicheck',
 		'taxonomy' => 'post_tag', // Taxonomy Slug
 		// 'inline'  => true, // Toggles display to inline
 	) );
 
 	$cmb_demo->add_field( array(
-		'name' => __( 'Test Checkbox', 'cmb2' ),
-		'desc' => __( 'field description (optional)', 'cmb2' ),
+		'name' => esc_html__( 'Test Checkbox', 'cmb2' ),
+		'desc' => esc_html__( 'field description (optional)', 'cmb2' ),
 		'id'   => $prefix . 'checkbox',
 		'type' => 'checkbox',
 	) );
 
 	$cmb_demo->add_field( array(
-		'name'    => __( 'Test Multi Checkbox', 'cmb2' ),
-		'desc'    => __( 'field description (optional)', 'cmb2' ),
+		'name'    => esc_html__( 'Test Multi Checkbox', 'cmb2' ),
+		'desc'    => esc_html__( 'field description (optional)', 'cmb2' ),
 		'id'      => $prefix . 'multicheckbox',
 		'type'    => 'multicheck',
 		// 'multiple' => true, // Store values in individual rows
 		'options' => array(
-			'check1' => __( 'Check One', 'cmb2' ),
-			'check2' => __( 'Check Two', 'cmb2' ),
-			'check3' => __( 'Check Three', 'cmb2' ),
+			'check1' => esc_html__( 'Check One', 'cmb2' ),
+			'check2' => esc_html__( 'Check Two', 'cmb2' ),
+			'check3' => esc_html__( 'Check Three', 'cmb2' ),
 		),
 		// 'inline'  => true, // Toggles display to inline
 	) );
 
 	$cmb_demo->add_field( array(
-		'name'    => __( 'Test wysiwyg', 'cmb2' ),
-		'desc'    => __( 'field description (optional)', 'cmb2' ),
+		'name'    => esc_html__( 'Test wysiwyg', 'cmb2' ),
+		'desc'    => esc_html__( 'field description (optional)', 'cmb2' ),
 		'id'      => $prefix . 'wysiwyg',
 		'type'    => 'wysiwyg',
 		'options' => array( 'textarea_rows' => 5, ),
 	) );
 
 	$cmb_demo->add_field( array(
-		'name' => __( 'Test Image', 'cmb2' ),
-		'desc' => __( 'Upload an image or enter a URL.', 'cmb2' ),
+		'name' => esc_html__( 'Test Image', 'cmb2' ),
+		'desc' => esc_html__( 'Upload an image or enter a URL.', 'cmb2' ),
 		'id'   => $prefix . 'image',
 		'type' => 'file',
 	) );
 
 	$cmb_demo->add_field( array(
-		'name'         => __( 'Multiple Files', 'cmb2' ),
-		'desc'         => __( 'Upload or add multiple images/attachments.', 'cmb2' ),
+		'name'         => esc_html__( 'Multiple Files', 'cmb2' ),
+		'desc'         => esc_html__( 'Upload or add multiple images/attachments.', 'cmb2' ),
 		'id'           => $prefix . 'file_list',
 		'type'         => 'file_list',
 		'preview_size' => array( 100, 100 ), // Default: array( 50, 50 )
 	) );
 
 	$cmb_demo->add_field( array(
-		'name' => __( 'oEmbed', 'cmb2' ),
-		'desc' => __( 'Enter a youtube, twitter, or instagram URL. Supports services listed at <a href="http://codex.wordpress.org/Embeds">http://codex.wordpress.org/Embeds</a>.', 'cmb2' ),
+		'name' => esc_html__( 'oEmbed', 'cmb2' ),
+		'desc' => sprintf(
+			/* translators: %s: link to codex.wordpress.org/Embeds */
+			esc_html__( 'Enter a youtube, twitter, or instagram URL. Supports services listed at %s.', 'cmb2' ),

```

Comment:
When you want to leave comments for translators, the `_x()` family of functions should be used. e.g.
```php
esc_html_x( 'Enter a youtube, twitter, or instagram URL. Supports services listed at %s.', '%s is a link to codex.wordpress.org/Embeds', 'cmb2' ),

```